packages/tasks/src/local-apps.ts

```diff
@@ -180,6 +183,34 @@ const snippetVllm = (model: ModelData): LocalAppSnippet[] => {
 		},
 	];
 };
+const snippetTgi = (model: ModelData): LocalAppSnippet[] => {
+	const runCommand = [
+		"# Call the server using curl:",
+		`curl -X POST "http://localhost:8000/v1/chat/completions" \\`,
+		`	-H "Content-Type: application/json" \\`,
+		`	--data '{`,
+		`		"model": "${model.id}",`,
+		`		"messages": [`,
+		`			{"role": "user", "content": "What is the capital of France?"}`,
+		`		]`,
+		`	}'`,
+	];
+	return [
+		{
+			title: "Use Docker images",
+			setup: [
+				"# Deploy with docker on Linux:",
+				`docker run --gpus all \\`,
+				`	-v ~/.cache/huggingface:/root/.cache/huggingface \\`,
+				` 	-e HF_TOKEN=<secret>" \\`,
+				`	-p 8000:80 \\`,
+				`	ghcr.io/huggingface/text-generation-inference:latest \\`,
```

Comment:
I think @Narsil is not very fond of encouraging the use of `latest` because it makes issues harder to track. But yes, using `2.3.1` would quickly become obsolete.

Reply:
Yeah! not sure what's the best way here

Reply:
`latest` is not the latest tagged release?

Reply:
But let's use latest for now (same as vllm)